.travis.yml

```diff
@@ -1,11 +1,27 @@
 language: cpp
-
+sudo: required
+dist: trusty
+os:
+  - linux
+  - osx
 compiler:
   - clang
   - gcc
+rvm:
+  - 1.9.3
+  - 2.0
+  - 2.2
```

Comment:
After all Ruby deps are there just to run our tests. We shouldn't be concerned with testing multiple Ruby versions. Let's just choose one thst works with rvm (.ruby-version).